### PR TITLE
Added errorPage, modified finalPage.jsp and ApplicationController.java

### PR DESCRIPTION
**Pull request by made-yudistira**

### CODE REVIEW

File: src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java [hunk 1]
```diff
@@ -11,11 +11,16 @@
 import org.springframework.web.bind.annotation.RequestMapping;
```
Comment (made-eka) [inline src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java:11]: For each Java class, please provide:
- class header
- Javadoc comment for each methods

File: src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java [hunk 2]
```diff
@@ -11,11 +11,16 @@
...
+/**
+ * The controller class for the Spring application.
+ * @author IDMAYUD
+ */
 @Controller
 public class ApplicationController {
  
 	private static final String INDEX_PAGE_KEY = "index";
 	private static final String FINAL_PAGE_KEY = "finalPage";
+	private static final String ERROR_PAGE_KEY = "errorPage";
```
Comment (made-eka) [inline src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java:23]: `private static final` is preferred

File: src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java [hunk 3]
```diff
@@ -11,11 +11,16 @@
...
 	private final String MESSAGE_MAP_KEY = "input_message";
 	private final static Logger logger = LoggerFactory.getLogger(ApplicationController.class);
```
Comment (made-eka) [inline src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java:25]: for `final` constant, the constant name should be all-capitalized

File: war/WEB-INF/pages/errorPage.jsp [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<c:url var="resources" value="/resources/theme1" scope="request" />
+
+<html>
+<head>
+<link href="${resources}/css/core.css" rel="stylesheet">
+</head>
+<body>
+	<abb_h1>This is Error Page!</abb_h1><br>	
+	<abb_h2>${input_message}</abb_h2>
+	<table>
```
Comment (made-eka) [inline war/WEB-INF/pages/errorPage.jsp:13]: please put `<table>` inside `<form>`

File: war/WEB-INF/pages/errorPage.jsp [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        <form:form method="GET" action="/ABBJavaDev-SpringMVC/toIndex">
+            <table>
+                <tr>
+                    <td><input type="submit" value="Back"/></td>
+			    </tr>
+			</table>  
+		</form:form>
+	</table>
```
Comment (made-eka) [inline war/WEB-INF/pages/errorPage.jsp:21]: Please adjust the indentations

File: war/WEB-INF/pages/finalPage.jsp [hunk 6]
```diff
@@ -9,5 +10,12 @@
     <body>
         <abb_h1>The Final Page</abb_h1><br>
         <abb_h2>${input_message}</abb_h2>
+        <form:form method="GET" action="/ABBJavaDev-SpringMVC/toIndex">
+            <table>
+                <tr>
+                    <td><input type="submit" value="Back"/></td>
+			    </tr>
+			</table>  
+		</form:form>
```
Comment (made-eka) [inline war/WEB-INF/pages/finalPage.jsp:19]: Please adjust the indentations

File: src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java [hunk 7]
```diff
@@ -44,5 +51,12 @@ public String finalPage(ModelMap model) {
```
Comment (kristiantoyanuar) [inline src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java:51]: Please do log the incoming request using the `logger`

File: src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java [hunk 8]
```diff
@@ -44,5 +51,12 @@ public String finalPage(ModelMap model) {
 		
 		return FINAL_PAGE_KEY;
 	}
+	
+	@RequestMapping(value = "/toIndex", method = RequestMethod.GET)
+	public String toIndexPage(ModelMap model) {
+		model.addAttribute(MESSAGE_MAP_KEY, "Welcome back!" );
+
```
Comment (kristiantoyanuar) [inline src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java:58]: Please do log the incoming request using the `logger`

**Pull request closed without merging**